sample/DateTimeParser/DateTimeParser.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using Superpower;
+using Superpower.Parsers;
+
+namespace DateTimeParser
+{
+    static public class DateTimeParser 
+    {
+        private static TextParser<string> _2DigitParser =
```

Comment:
`TwoDigits` might be a friendlier name?

It'd be nice if we provided `Repeat(exact)`  and `Repeat(min, max)` combinators, so this could be written as:

```csharp
Character.Digit.Repeat(2).Select(chs => new string(chs))
```

Sprache (upon which Superpower is based) provides a `.Text()` combinator for the common `IEnumerable<char>` &rarr; `string` conversion, so this could be further improved to:

```csharp
Character.Digit.Repeat(2).Text()
```